installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -129,7 +129,7 @@ If you choose to create each rule based on events, you must create firewall rule
 .Example output
 [source,terminal]
 ----
-Firewall change required by security admin: `gcloud compute firewall-rules create k8s-fw-a26e631036a3f46cba28f8df67266d55 --network example-network --description "{\"kubernetes.io/service-name\":\"openshift-ingress/router-default\", \"kubernetes.io/service-ip\":\"35.237.236.234\"}\" --allow tcp:443,tcp:80 --source-ranges 0.0.0.0/0 --target-tags exampl-fqzq7-master,exampl-fqzq7-worker --project example-project`
+./oc get events -n openshift-ingress --field-selector="reason=LoadBalancerManualChange" Firewall change required by security admin: `gcloud compute firewall-rules create k8s-fw-a26e631036a3f46cba28f8df67266d55 --network example-network --description "{\"kubernetes.io/service-name\":\"openshift-ingress/router-default\", \"kubernetes.io/service-ip\":\"35.237.236.234\"}\" --allow tcp:443,tcp:80 --source-ranges 0.0.0.0/0 --target-tags exampl-fqzq7-master,exampl-fqzq7-worker --project example-project`
```

Comment:
+1 to @tsze-redhat's other related comment about a line break and dropping the `./`. But further, if this addition is intending to illustrate what command you could run to view this example output, it seems like the command should also be in its own code block (with a `$` prompt), prior to the example output block, to conform to our normal guidelines on showing commands with output. For example:

```
[source,terminal]
----
$ oc get events -n openshift-ingress --field-selector="reason=LoadBalancerManualChange"
---

.Example output
[source,terminal]
----
Firewall change required by security admin: `gcloud compute firewall-rules create k8s-fw-a26e631036a3f46cba28f8df67266d55 --network example-network --description "{\"kubernetes.io/service-name\":\"openshift-ingress/router-default\", \"kubernetes.io/service-ip\":\"35.237.236.234\"}\" --allow tcp:443,tcp:80 --source-ranges 0.0.0.0/0 --target-tags exampl-fqzq7-master,exampl-fqzq7-worker --project example-project`
----
```

Maybe also with a lead-in sentence before the `oc get events` command to provide some context for what's being shown. But if I'm misunderstood and the literal example output includes the `oc get events`, then disregard.